### PR TITLE
Added country field

### PR DESCRIPTION
**Pull request by irfanuddinahmad**
**Description:** 
Added country field to EnterpriseCustomer model

**JIRA:** 
https://openedx.atlassian.net/browse/ENT-1197

**Testing instructions:**

1. Open page http://business.sandbox.edx.org/admin/enterprise/enterprisecustomer/
2. Verify country field
3. Create new enterprise customer
4. Verify that country is stored

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Core requirements for using this application that are not provided by edx-platform
 
+django-countries==4.6.1
```
Comment (zubair-arbi) [inline requirements/base.in]: We might not want to put it here as the description of this file states:
```
Core requirements for using this application that are not provided by edx-platform
```

File: enterprise/models.py [hunk 2]
```diff
@@ -126,6 +127,7 @@ class Meta:
         help_text=_("Course catalog for the Enterprise Customer.")
     )
     active = models.BooleanField(default=True)
+    country = CountryField(null=True)
```
Comment (zubair-arbi) [inline enterprise/models.py]: We missed some of the AC's:

1. Required field
2. Default for current customers to US

Comment (irfanuddinahmad) [reply enterprise/models.py]: @zubair-arbi The field is currently 'required' and CS will update the current customers to US

> We missed some of the AC's:
> 
> 1. Required field
> 2. Default for current customers to US

**Comment by georgebabey** [comment]
Looks good. I was expecting non-null in the db with a migration to set a default for current value, but since `blank` is not set on the field this works to enforce it in django-admin screens

**Pull request merged**